.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea/
```

Comment:
`<random-thought>`
Random comment having nothing to do with this class nor even javascript: you can define a global `gitignore` file that applies to all repositories, which is a good place for stuff like this. I [do it that way](https://github.com/joshdmiller/dot-files/blob/master/home/gitconfig) so I can omit files that my editor generates without (a) having to change every gitignore in every repo I ever touch; and (b) to prevent committing irrelevant code to repos.
`</random-thought>`